addons/gamification/views/gamification_karma_tracking_views.xml

```diff
@@ -5,12 +5,22 @@
         <field name="model">gamification.karma.tracking</field>
         <field name="arch" type="xml">
             <search string="Search Trackings">
-                <field name="user_id"/>
+                <field name="user_id" string="Karma Owner"/>
                 <field name="tracking_date"/>
+                <field name="origin_type_id"/>
                 <filter string="Consolidated" name="filter_consolidated"
                     domain="[('consolidated', '=', True)]"/>
+                <filter string="My Karma" name="filter_user_id"
+                    domain="[('user_id', '=', uid)]"/>
+                <separator/>
+                <filter string="Manual" name="filter_res_users"
+                    domain="[('origin_ref', 'ilike', 'res.users,')]"/>
```

Comment:
Not sure I understand this one?

Reply:
I meant what is the link between "Manual" and the origin being "res.users" ?

I guess it's to differentiate it from changes from course completion etc, wording is a bit strange, but if it works it works.